Assets/MixedRealityToolkit.SDK/Experimental/Features/UX/BoundsControl/BoundsControl.cs

```diff
@@ -192,39 +192,39 @@ public Vector3 BoxPadding
 
         [SerializeField]
         [Tooltip("Bounds control box display configuration section.")]
-        private BoundsControlBoxDisplay boxDisplay = new BoundsControlBoxDisplay();
+        private BoundsControlBoxDisplay boxDisplay;
         /// <summary>
         /// Bounds control box display configuration section.
         /// </summary>
         public BoundsControlBoxDisplay BoxDisplay => boxDisplay;
 
         [SerializeField]
         [Tooltip("This section defines the links / lines that are drawn between the corners of the control.")]
-        private BoundsControlLinks links = new BoundsControlLinks();
+        private BoundsControlLinks links;
```

Comment:
Suggestion/Consideration:

Put all of these into a BoundsControlConfigProfile or BoundsControlProfile etc. 

Reasoning:

BoundsCOntrol is one of those UX where there is a 50/50 chance this is being created via a prefab/scene or being added dynamically to an object. In the later half, I would like to properly add a pre-configured BoundsControl. It would be awesome to be able to do 

var boundsControl = myGameObject.AddComponent<BoundsCOntrol>();
boundsControl.ApplyConfiguration(mySavedProfile);
or
boundsControl.ConfigProfile = mySavedProfile; etc

Reply:
I'm not sure if we really want to do that - it would add another layer on top of the other visuals - if we combine them into one config then you'll lose the ability to share single visual elements - if i leave them apart and add the layer on top it's more cumbersome to set up. I'd rather not have the user create and link 2 assets to be able to change one property in a eg rotation handle :-/